src/Assets/WasmOverride/Nuget.config

```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <configuration>
+  <add key="signatureValidationMode" value="accept" />
```

Comment:
Is this change temporary?

Reply:
No, this is the same as other tests on the repo, this config didn't have it.